src/base.css

```diff
@@ -1344,6 +1346,21 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Character examples (from the I18N template) *******************************/
+.codepoint bdi {
```

Comment:
If this is supposed to handle multi-character sequences, should it be .unicode instead of .codepoint?

Reply:
Good point. Perhaps `.codepoints`?

Reply:
Changing the class name will break a lot of our markup ☹️ so we decided to leave as-is

Reply:
In what way?

Reply:
We have many many documents with the class `codepoint` in it. If we rename the class to `codepoints` we'd need to change the markup to match. Of course we could carry a local style of `codepoint` while doing migration......

Ultimately our goal is to automate this so that people don't type in the template but instead generate it from a more compact markup using features in e.g. Bikeshed or ReSpec.